packages/edit-widgets/src/blocks/widget-area/index.js

```diff
@@ -16,5 +16,6 @@ export const settings = {
 	title: __( 'Widget Area' ),
 	description: __( 'A widget area container.' ),
 	__experimentalLabel: ( { name: label } ) => label,
+	hideParentSelector: true,
```

Comment:
I think we should make this `experimental` to start with, and consider moving it to the `supports` object:
https://developer.wordpress.org/block-editor/developers/block-api/block-supports/

Something like this where it defaults to `true` might be an option:
```js
supports: {
    __experimentalParentSelector: false,
}
```
There's then a `hasBlockSupport` function that can be used.

Worth keeping an eye on this PR as well:
https://github.com/WordPress/gutenberg/pull/23800

Reply:
Although the naming is a bit funny, as it's really the child block that you can't select the parent from. Making it experimental gives us the capability of changing the API in the future.

Reply:
Sounds good to me! Just pushed 3075c84.